data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/export/ManifestFileReader.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.dynamodb.export;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.opensearch.dataprepper.plugins.source.dynamodb.model.ExportSummary;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ManifestFileReader {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ManifestFileReader.class);
+
+    private static final String DATA_FILE_S3_KEY = "dataFileS3Key";
+    private static final String DATA_FILE_ITEM_COUNT_KEY = "itemCount";
+
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    private final S3ObjectReader fileReader;
+
+    public ManifestFileReader(S3ObjectReader fileReader) {
+        this.fileReader = fileReader;
+    }
+
+    public ExportSummary parseSummaryFile(String bucket, String key) {
+        LOG.debug("Try to read the manifest summary file");
```

Comment:
Perhaps include the key here to make this a more useful debug log line.

Reply:
The detailed debug log is the S3ObjectReader its using (with bucket and key), so here is just a general info so we know what the read is for.